tests/pages/main_page_locators.py

```diff
@@ -0,0 +1,49 @@
+from selene import by
+
+
+class LogInPopupLocators:
+    CONFIRMATION_LOG_IN_BUTTON = (by.xpath("//form[@data-ref='login_modal']//button[@type='submit']"))
+    EMAIL_FORM = (by.xpath("//input[@placeholder='email@email.com']"))
+    PASSWORD_FORM = (by.xpath("//input[@placeholder='Password']"))
+
+
+class MainPageLocators:
+    ACCEPT_COOKIES_BUTTON = (by.xpath("//div[@id='cookie-popup-with-overlay']//button"
+                                      "[@class='cookie-popup-with-overlay__button']")
+                             )
```

Comment:
consider using triple quotes if it is needed

Reply:
Fixed